codegen/masm/src/stackify/emitter.rs

```diff
@@ -721,7 +721,7 @@ impl<'a> MasmEmitter<'a> {
         };
         let pos = stack
             .find(&value)
-            .expect("value not found on operand stack");
+            .expect(format!("value {} not found on operand stack", value).as_str());
```

Comment:
I wonder if it would be more convenient for us to modify `find` to raise a pretty error like this if the value isn't found. I should check and see if we ever handle the `None` case with anything other than a panic. We don't need to do that here, but if you have thoughts on it, let me know.

Reply:
Sure, that sounds great to me.